error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -263,9 +280,12 @@ boolean after(Stream<T> stream) {
 
   static final class StreamMin<T> {
     @BeforeTemplate
+    @SuppressWarnings("java:S1155" /* This violation will be rewritten. */)
```

Comment:
Here and below: we _could_ introduce a separate method to further reduce this suppression's scope, but not sure that this is worth it.